packages/medusa/src/api/routes/store/products/search.ts

```diff
@@ -2,6 +2,7 @@ import { Type } from "class-transformer"
 import { IsNumber, IsOptional, IsString } from "class-validator"
 import { SearchService } from "../../../../services"
 import ProductService from "../../../../services/product"
+import { PriceSelectionParams } from "../../../../types/price-selection"
```

Comment:
To avoid that I suggest to setup the `paths` in the tsconfig can see [here](https://github.com/medusajs/medusa/pull/1254)